go.mod

```diff
@@ -1,9 +1,9 @@
-module github.com/nspcc-dev/neofs-api-go/v2
+module github.com/TrueCloudLab/frostfs-api-go/v2
 
 go 1.17
 
 require (
-	github.com/nspcc-dev/neofs-crypto v0.4.0
+	github.com/TrueCloudLab/frostfs-crypto v0.5.0
```

Comment:
Push this tag. All other tags were removed from frostfs-crypto.

Reply:
Fixed